frontend/src/metabase/query_builder/components/DataSelector/TriggerComponents.tsx

```diff
@@ -80,7 +80,12 @@ export function FieldTrigger({
 
 export function DatabaseTrigger({ database }: { database: Database }) {
   return database ? (
-    <span className="text-wrap text-grey no-decoration">{database.name}</span>
+    <span
+      className="text-wrap text-grey no-decoration"
+      data-testid="selected-database"
```

Comment:
Yes, it is the same because they can never be rendered at the same time. It's either-or type of situation, but the meaning of that element is the same - it shows the selected database.

1. when it's the only option and thus rendered by default
2. when it's the chosen option out of many